lib/webpacker/configuration.rb

```diff
@@ -25,33 +25,37 @@ def public_path
     end
 
     def config_path
-      Rails.root.join(fetch(:config))
+      Rails.root.join(paths[:config])
     end
 
     def file_path(root: Rails.root)
-      root.join("config/webpack/paths.yml")
+      root.join("config/webpacker.yml")
     end
 
     def default_file_path
       file_path(root: Pathname.new(__dir__).join("../install"))
     end
 
     def source
-      fetch(:source)
+      paths[:source]
     end
 
-    def fetch(key)
-      paths.fetch(key, default_paths[key])
-    end
```

Comment:
Before, if `paths.output` was missing in my `webpacker.yml` file, it would use the default. Now it will only use the default if `paths` is entirely missing.